spr4.js

```diff
@@ -0,0 +1,50 @@
+
+class Book {
+    constructor(title, _author) {
+        this.title = title; // public
```

Comment:
Don't make data fields public. Let's make it protected or private

Reply:
Thanks for the review. I created public data because it was in the task